openff/toolkit/utils/rdkit_wrapper.py

```diff
@@ -217,8 +217,8 @@ def from_pdb_and_smiles(
         # check isomorphic and get the mapping if true the mapping will be
         # Dict[pdb_index: offmol_index] sorted by pdb_index
```

Comment:
```suggestion
        # Dict[offmol_index, pdbmol_index] sorted by offmol index
```
I think this docstring is out of date? I'm trying to figure out the behavior change of this diff and am somewhat confused - now when atom ordering in the PDB file and SMILES disagree, the SMILES wins (by nature of the `Molecule` derived from SMILES being first in this method call)?

Reply:
Yeah that comment is out of date, thanks for catching that! I think [Jeff's explanation](https://github.com/openforcefield/openff-toolkit/pull/1400#discussion_r971550949) is exactly right - the previous behavior was incorrect, and the new behavior is the previously documented behavior (PDB wins).

Reply:
So the resulting atom ordering is NEITHER the atom ordering in the PDB file, the SMILES (as processed by RDKit + OpenFF), but instead a third atom ordering? Am I totally misunderstanding what remapping?

Reply:
I'm not sure I'm understanding you - the behavior prior to this PR is that the atoms are given a third ordering, but the coordinates are assigned as though the PDB order prevailed. This meant that we violated a guarantee we made in the docstring, AND that the coordinates were incorrect. This PR fixes that bug so that the coordinates are correct and the PDB ordering prevails.

Reply:
Okay, I misread.